help/markdown/fake-template.md

```diff
@@ -40,6 +40,12 @@ Specifies your prefered way to define the nuget packages used in your build:
 - `inline` - Defines build dependencies inside the build script
 - `none` - Use this if you already have a `paket.dependencies` in your folder
 
+### --dsl
+Specifies your prefered way to define build tasks inside your build script:
+
+- `fake` (default) - Uses the default FAKE domain specific language
+- `blackfox` - Uses the BlackFox domain specific language
```

Comment:
I would add a link to the docs here.

Reply:
Do you mean a link to the docs for each dsl?

Reply:
Yes, i think that would be convenient, especially for the blackfox one.